File: client/src/model/GeneralDataSection.ts
interface IGeneralDataSection {
	id: number
	name: string
    orderNum: number
}

class GeneralDataSection {
	id: number
	name: string
    orderNum: number

	constructor(obj?: IGeneralDataSection) {
		this.id = (obj && obj.id) || 0
		this.name = (obj && obj.name) || ''
		this.orderNum = (obj && obj.orderNum) || 0
	}
}

export default GeneralDataSection

File: server/DocumentsKM/Models/GeneralDataSection.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace DocumentsKM.Models
{
    // Раздел общих указаний
    public class GeneralDataSection
    {
        [Key]
        public Int16 Id { get; set; }

        // Название
        [Required]
        [MaxLength(255)]
        public string Name { get; set; }

        // Номер
        [Required]
        public Int16 OrderNum { get; set; }

        // Пункты общих указаний
        public virtual IList<GeneralDataPoint> GeneralDataPoints { get; set; } = new List<GeneralDataPoint>();
    }
}
